src/AzureDataFactory.TestingFramework.Example/BatchJob/BatchJobUnitTests.cs

```diff
@@ -66,8 +66,8 @@ public void SetManagerApplicationPackagePathTest()
         // Arrange
         var activity = _pipeline.GetActivityByName("Set ManagerApplicationPackagePath") as SetVariableActivity;
         var variable = new PipelineRunVariable<string>("ManagerApplicationPackagePath");
-        _state.Parameters.Add(new RunParameter<string>(ParameterType.Parameter, "ManagerApplicationPackageName", "managerworkload"));
-        _state.Parameters.Add(new RunParameter<string>(ParameterType.Parameter, "ManagerApplicationPackageVersion", "0.13.2"));
```

Comment:
Could we create a unique variable that holds the versions of the ManagerApplicationPackageVersions for the different package names. The idea is if the version changes it doesn't need to be changed on all different possible  places is being used.

Reply:
It's just an example value and not being used by anything. Will leave this for now :)